### PR TITLE
Added a reference to Scapy topic.

### PR DESCRIPTION
**Pull request by joesinghh**
I have added an reference to scary topic under MISC section as asked [here](https://github.com/secdev/scapy/pull/3382#issuecomment-932888168)

### CODE REVIEW

**Comment by joesinghh** [comment]
@guedou could you please review it. Thank you.

File: README.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ Misc
 - [scapy-benchmarks](https://github.com/gpotter2/scapy-benchmarks): a small test suite that tracks the evolution of Scapy's performance.
 - [sshame](https://github.com/HynekPetrak/sshame): tool to brute force SSH public-key authentication
 - [TIDoS Framework](https://github.com/0xInfection/TIDoS-Framework): The Offensive Manual Web Application Penetration Testing Framework
+- [scapy-topic](https://github.com/topics/scapy): Scapy topic on GitHub explore.
```
Comment (guedou) [inline README.md]: I will rather have:
```
- [Explore Scapy topics](https://github.com/topics/scapy) on GitHub
```

**Review by guedou — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (joesinghh) [reply README.md]: done 👍🏻

**Comment by joesinghh** [comment]
@guedou could you please merge it as `github/explore` has merged my PR.

**Pull request merged**